7-layouts/styles.css

```
@@ -0,0 +1,145 @@
+/*Mobile design*/
+.articles {
+    display: flex;
+    flex-flow: column wrap;
+    margin-top: 40px;
+}
+
+.articles .animal {
+    display: flex;
+    flex-direction: column-reverse;
+    height: auto;
+    margin-bottom: 35px;
+}
+
+.articles .animal .animal-img {
+    display: block;
+    height: auto;
+    width: 100%;
+}
+
+.articles .animal .animal-title {
+    background-color: rgba(88, 88, 88, 0.281);
+    font-size: 25px;
+    font-weight: 10;
+    position: absolute;
+    margin: 0;
+    width: 100%;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.header .title-section:first-child {
+    align-items: center;
+    background-color: #927811;
+    display: flex;
+    height: 40px;
+    padding-left: 10px;
+}
+
+.header .title-section:first-child .page-title {
+    color: white;
+    margin-left: 20px;
+}
+
+.header .title-section:first-child .title-img {
+    height: 30px;
+    width: 30px;
+}
+
+.header .welcome-and-img-section:nth-child(2) {
+    background: url("assets/main-picture.jpg");
+    background-position: center center;
+    background-size: cover;
+    color: white;
+    font-family: Calibri;
+    height: 300px;
+    text-align: justify;
+}
+
+.header .welcome-and-img-section:nth-child(2) .site-desc {
+    font-size: 20px;
+}
+
+.header .welcome-and-img-section:nth-child(2) .welcome-msg {
+    font-size: 50px;
+    margin: 0;
+}   
+
+.header .welcome-and-img-section:nth-child(2) .welcome-msg, .header .welcome-and-img-section:nth-child(2) .site-desc {
```

```suggestion
.description {
```
and in the html you can use the same class for both elements, these selectors are complex and unnecessary.